Added blog post about CI. (#1094)
USRSE/usrse.github.io: _posts/2023-03-21-announcing-ci.md

---
layout: post

> title: "US-RSE has a new Fiscal Sponsor!"

Reviewer: ```suggestion
title: "US-RSE Has a New Fiscal Sponsor!"
```

Reviewer: I realize this is nitpicky, but the mixed capitalization looks weird to me!

Author: Usually, I'm the one making those nitpicky changes, so thanks! 😆

tags: [finances]
author: Julia Damerow
---

We’re excited to announce that US-RSE has switched to a new fiscal sponsor. We are now a fiscally sponsored project of [Community Initiatives](https://communityin.org/) (CI)! Our previous fiscal sponsor, Open Collective Foundation (OCF), represented an important first step allowing us to accept donations and making it possible to reimburse members for expenses they incurred on behalf of US-RSE. 
Moving forward, Community Initiatives’ additional tools, resources, and infrastructure will better serve US-RSE’s future organizational and financial goals.

[According to Wikipedia](https://en.wikipedia.org/wiki/Fiscal_sponsorship), “[f]iscal sponsorship refers to the practice of non-profit organizations offering their legal and tax-exempt status to groups—typically projects—engaged in activities related to the sponsoring organization's mission.” 

> For US-RSE, that means that by being a project of Community Initiatives, their 501(c)(3) non-profit status extends to us, so that, for instance, we can accept donations without the expense or effort of having to form a non-profit ourselves. 

Reviewer 2: ```suggestion
For US-RSE, that means that by being a project of CI, their 501(c)(3) non-profit status extends to us, so that, for instance, we can accept donations without the expense or effort of having to form a non-profit ourselves. 
```

Furthermore, CI also provides services such as legal consultancy and HR services that allow us to enter into contracts or hire employees. 
Because we are legally a project of CI, however, US-RSE will need to abide by federal and California laws and regulations pertaining to 501(c)(3) organizations. 

By becoming a fiscally sponsored project of CI, US-RSE now has access to services such as grant support (we will be able to accept grants from federal and private funders), legal services, support for crowdfunding and donation management, as well as certain types of insurance for volunteers. 
CI charges 10% of gross revenues, and 15% of received government funds, which is in line with most other fiscal sponsors but much lower than typical academic institutions. 
US-RSE carefully evaluated the different options of fiscal sponsorship and concluded that CI was the best choice. 
The agreement between US-RSE and CI is not permanent. 

> Therefore, the option for US-RSE to become its own 501(c)(3) non-profit organization is still possible, though we feel strongly that, at this point, it would be too expensive and time consuming.

Reviewer 3: ```suggestion
US-RSE could still become its own 501(c)(3) non-profit organization in the future, though we feel strongly that it would be too expensive and time consuming at this point.
```

Reviewer 3: simpler language

> We look forward to the exciting new possibilities and opportunities for US-RSE that are now possible as a fiscally sponsored project of Community Initiatives! 

Reviewer 2: ```suggestion
We look forward to the exciting new opportunities for US-RSE that are now possible as a fiscally sponsored project of Community Initiatives! 
```
